includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -687,37 +803,81 @@ public function wc_setup_shipping_save() {
+				'class'       => $can_stripe ? 'checked' : '',
+				'repo-slug'   => 'woocommerce-gateway-stripe',
+				'settings'    => array(
+					'email' => array(
+						'label'       => __( 'Stripe email address', 'woocommerce' ),
+						'type'        => 'email',
+						'value'       => get_option( 'admin_email' ),
+						'placeholder' => __( 'Stripe email address', 'woocommerce' ),
+					),
+				),
+				'enabled' => $can_stripe,
+				'featured' => true,
+			),
 			'braintree_paypal' => array(
 				'name'        => __( 'PayPal by Braintree', 'woocommerce' ),
 				'image'       => WC()->plugin_url() . '/assets/images/paypal-braintree.png',
-				'description' => __( "Safe and secure payments using credit cards or your customer's PayPal account.", 'woocommerce' ) . ' <a href="https://wordpress.org/plugins/woocommerce-gateway-paypal-powered-by-braintree/" target="_blank">' . __( 'Learn more about PayPal', 'woocommerce' ) . '</a>',
-				'class'       => 'featured featured-row-last',
+				'description' => __( "Safe and secure payments using credit cards or your customer's PayPal account.", 'woocommerce' ) . ' <a href="https://wordpress.org/plugins/woocommerce-gateway-paypal-powered-by-braintree/" target="_blank">' . __( 'Learn more about PayPal', 'woocommerce' ) . '</a>.',
+				'class'       => 'in-cart',
 				'repo-slug'   => 'woocommerce-gateway-paypal-powered-by-braintree',
 			),
 			'ppec_paypal' => array(
 				'name'        => __( 'PayPal Express Checkout', 'woocommerce' ),
 				'image'       => WC()->plugin_url() . '/assets/images/paypal.png',
 				'description' => __( "Safe and secure payments using credit cards or your customer's PayPal account.", 'woocommerce' ) . ' <a href="https://wordpress.org/plugins/woocommerce-gateway-paypal-express-checkout/" target="_blank">' . __( 'Learn more about PayPal', 'woocommerce' ) . '</a>',
-				'class'       => 'featured featured-row-last',
+				'class'       => 'in-cart',
 				'repo-slug'   => 'woocommerce-gateway-paypal-express-checkout',
 			),
-			'stripe' => array(
-				'name'        => __( 'Stripe', 'woocommerce' ),
-				'image'       => WC()->plugin_url() . '/assets/images/stripe.png',
-				'description' => sprintf( __( 'A modern and robust way to accept credit card payments on your store. <a href="%s" target="_blank">Learn more about Stripe</a>.', 'woocommerce' ), 'https://wordpress.org/plugins/woocommerce-gateway-stripe/' ),
-				'class'       => 'featured featured-row-first',
-				'repo-slug'   => 'woocommerce-gateway-stripe',
-			),
 			'paypal' => array(
 				'name'        => __( 'PayPal Standard', 'woocommerce' ),
 				'description' => __( 'Accept payments via PayPal using account balance or credit card.', 'woocommerce' ),
 				'image'       => '',
```

Comment:
No image?

Reply:
This is unchanged from the current wizard: https://github.com/woocommerce/woocommerce/blob/master/includes/admin/class-wc-admin-setup-wizard.php#L719